toplevel/usage.ml

```diff
@@ -32,6 +32,7 @@ let print_usage_channel co command =
 \n  -R dir coqdir          recursively map physical dir to logical coqdir\
 \n  -Q dir coqdir          map physical dir to logical coqdir\
 \n  -top coqdir            set the toplevel name to be coqdir instead of Top\
+\n  -topfile f             set the toplevel name as though compiling f\
```

Comment:
typo, "though" is not what you want to say. I'd say "as if it was compiling f"

Reply:
Doesn't seem to be actually different https://en.wiktionary.org/wiki/as_though
>as though
    As to suggest the idea that; as if, as would be true if.

Reply:
Oh, I learnt a thing ;-)